Resources/Audio/_Harmony/Lobby/attributions.yml

```diff
@@ -1,4 +1,9 @@
 ﻿- files: ["rustycansbattle.ogg"]
   license: "CC-BY-NC-SA-3.0"
   copyright: "Aaron Opfer, Rusty Cans Battle, created for FTL Captain's edition mod."
-  source: "https://www.subsetgames.com/forum/viewtopic.php?f=11&t=15663"
\ No newline at end of file
+  source: "https://www.subsetgames.com/forum/viewtopic.php?f=11&t=15663"
+
+﻿- files: ["harmonilla.ogg"]
+  license: ""
+  copyright: ""
+  source: ""
```

Comment:
I've been listening this on repeat... This is awesome but unhealthy!

@meatballwalkin I've commited an empty attributions structure for you to fill out.

* `license` field -- your chosen license. [CC-BY-NC-SA-3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/deed.en) is a good choice. If you want to explore more options, let me know on discord. I'll give you a list of supported licenses and their short descriptions.
* `copyright` field -- your name, name you use to publish music, twitter handle, discord handle, nick, email, whatever you wish. But something that IDs you.
* `source` -- your personal website, github profile link, or just harmony github URL if you it was never published by you anywhere else and you don't plan to do that. Your call which link to put in.

Hope this helps! Let me know on Discord if you need any help or have any questions!